services/director/producer.py

```diff
@@ -133,8 +133,9 @@ def start_service(service_name, service_tag, service_uuid):
     check_service_uuid_available(dockerClient, service_uuid)
     login_docker_registry(dockerClient)
 
-    # create a new network to connect the differnt containers
-    docker_network = create_overlay_network_in_swarm(dockerClient, service_name, service_uuid)
+    if len(listOfImages) > 1:
+      # create a new network to connect the differnt containers
+      docker_network = create_overlay_network_in_swarm(dockerClient, service_name, service_uuid)
```

Comment:
I am surprised to see that python linter is passing! ``dockerClient, listOfNames, dockerImagePath``... are non compliant. 
Is pylint actually running in travis?

Reply:
Definitely running "pylint ./services" using .pylintrc rules.
I will check if it's actually really checking all py files

Reply:
Yeah, I saw the log :) but it does not seem to be running

Reply:
pylint does not seem to work.
I renamed all the variables i could find that were camelcase.

Reply:
i will check pylint configuration  ...